### DISCUSSION TITLE
Where is the documentation about the webcam camera

### DISCUSSION THREAD

**Discussion by bobwatcherx** [widgetti/solara#233] [Q&A]
I'm looking for documentation on how to use a webcam on solara.. is there any. And if not, can you provide an example of the code?

**Comment by maartenbreddels** [comment]
There is https://github.com/maartenbreddels/ipywebrtc which can be used (see https://solara.dev/docs/howto/ipywidget-libraries)

Maybe we can make a nice example on how to use it. Do you have a particular use case in mind?

**Comment by bobwatcherx** [comment]
i want to integrate opencv like face recognition , object detection , face expression , more

**Comment by bobwatcherx** [comment]
can you give example code and add to docs
